javascript/ecmascript-2015/intro-ecmascript/ecmascript-6.md

```diff
@@ -3,6 +3,13 @@ author: alexjmackey
 
 levels:
   - beginner
+  - basic
+  - intermediate
+  - advanced
+
+aspects:
+  - introduction
+  - new
```

Comment:
ECMAScript 2015 was launched in June 2016, we're now on ECMAScript 2018 officially and ECMAScript2019 is finalized as well so this aspect might not apply anymore

Reply:
I left this comment before reading the changes bellow, looks like everything is getting generalized so I'll wait until you're done to ask questions :)

Reply:
I updated the insight- doesn't refer to that anymore. I need to change the name of the workout.

I reworked it to talk about the evolving specification, and how that specification relates to the different places JS is used and where that specification is implemented. I also linked to some compatibility tables on the first workout- a nice-to-have next step would be to link every ECMAScript insight to it's corresponding CanIUse.com and node.green section, which I'll file as an issue. 
I think those changes will make this curriculum more robust / future-proof as it will be easy enough to check to see whether or not we should consider the topic "New".

Reply:
Heh, it's the name of the _course_ I need to change. I'm going to leave the folder named like it is for now, I don't recall if renaming a course is as simple as renaming a workout.